src/unity/toolkits/drawing_recognition/data_preparation.hpp

```diff
@@ -0,0 +1,40 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+#ifndef TURI_DRAWING_DATA_PREPARATION_H_
+#define TURI_DRAWING_DATA_PREPARATION_H_
+
+#include <export.hpp>
+#include <unity/lib/gl_sframe.hpp>
+#include <unity/lib/variant.hpp>
+#include <unity/lib/image_util.hpp>
+
+namespace turi {
+namespace drawing_recognition {
+
+/**
+ *  This function is responsible for converting stroke-based drawing data
+ *  to 28x28 bitmaps to make it ready for training with the Neural Network.
+ *  
+ * 
+ * \param[in] data 					SFrame from the user, which contains 
```

Comment:
I don't think we need that much whitespace between the parameter name and its definition

Should we document here the expected input format of the SFrame?